### ISSUE TITLE
[Request] In DataTable, separate the "no data" view from the pager

### ISSUE THREAD

**Issue by aandr** [quasarframework/quasar#2075]
I'm using DataTable in several locations and I don't need or use the pager on the bottom, so I set the
`hide-bottom` flag. However, turning off the pager also hides the "No data available" view.

I think it's better if "No data available" still works even if the pager is disabled. While I understand, the two widgets may reuse the same container, functionally they are different components. If people need to disable the "no data" view, perhaps that should be a separate attribute for the `<q-table>` tag.

A temporary workaround is to have a conditional `hide-bottom` like so:

```
<q-table :data="myData" ... :hide-bottom="myData.length != 0" />
```

**Comment by marcelo-rebello** [comment]
I believe that is also related that the `no-data` scopedSlot is also not working when you have `hide-bottom`
The default no-data message is displayed ignoring the slot

**Comment by DelaneDiamon** [comment]
https://github.com/quasarframework/quasar/blob/dev/ui/src/components/table/table-bottom.js

Table bottom is using "no records" display.
Sad thing is - if you will not hide the bottom, but try to replace pagination slot with empty content, that will only get rid of the pagination arrows and record count, and still show "Rows per page"

**Comment by rstoenescu** [comment]
New Boolean props for bottom layer: hide-selected-banner, hide-no-data, hide-pagination
Will be available in "quasar" v1.12.0.
